libmultilabel/networks/kim_cnn.py

```diff
@@ -6,12 +6,18 @@
 
 
 class KimCNN(BaseModel):
-    def __init__(self, config, embed_vecs):
-        super(KimCNN, self).__init__(config, embed_vecs)
-
-        self.filter_sizes = config.filter_sizes
+    def __init__(
+        self,
+        embed_vecs,
+        num_classes,
+        filter_sizes=[2, 4, 8],
```

Comment:
Using mutable objects as the default value is somehow risky in Python.
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/
Let's set the default value as `None` and assign the value inside the function.

```suggestion
        filter_sizes=None,
```